examples/src/main/r/RSparkSQLExample.R

```diff
@@ -114,7 +114,7 @@ write.df(namesAndAges, "namesAndAges.parquet", "parquet")
 
 
 # $example on:manual_load_options_csv$
-df <- read.df("examples/src/main/resources/people.csv", "csv")
+df <- read.df("examples/src/main/resources/people.csv", "csv", sep=";", inferSchema=T, header=T)
```

Comment:
Hi, @felixcheung .
Could you review this?

Reply:
Also, ping @jomach and @gatorsmile because it was added by the following PR at Spark 2.3.
- https://github.com/apache/spark/pull/19429/files#diff-eeffb959b904ebb5c864bc3dafe6437dR117

BTW, [SPARK-20055](https://issues.apache.org/jira/browse/SPARK-20055) is still open.

Reply:
in R style we typical put space after param name, ie. https://github.com/apache/spark/pull/22791/files#diff-eeffb959b904ebb5c864bc3dafe6437dR168
`, sep = ";", inferSchema = TRUE, header = TRUE`

and pls don't use `T` for readability

Reply:
Thank you, @felixcheung .

Reply:
If you don't mind, I included that [here](https://github.com/apache/spark/pull/22801/files#diff-eeffb959b904ebb5c864bc3dafe6437dR117)